omar_qaterge.js

```diff
@@ -0,0 +1,202 @@
+//1
+console.log('Hello, World!');
+console.log('Ciao, mondo!');
+console.log('Salut, monde!');
+
+
+
+
+//2
+//problem:
+//console.log('I'm awesome');
+
+//Solution:
+console.log("I'm awesome");
+
+
+
+
+// 3
+let x;
+console.log("the value of my variable x will be: undefined");
+console.log(x);
+x = 69;
+console.log("the value of x will be: 69");
+console.log(x);
+
+
+
+
+//4
+let y = "JUST DO IT!";
+console.log("the value of my string will be: " + y);
+console.log(y);
+y = "maybe later...";
+console.log("the value of my string will be: " + y);
+console.log(y);
+
+
+
+
+//5
+//method 1
+let z = 7.25;
+console.log(z);
+const a = Math.round(z);
+console.log(a);
+const maxvalue = Math.max(a, z);
+console.log("the highest value of the two is: " + maxvalue);
+
+
+
+
+//method 2
+let z = 7.25;
+console.log(z);
+const a = Math.round(z);
+console.log(a);
+let highest_value;
+if (z > a) { 
+	highest_value = z
+    console.log("the highest value of the two is: " + highest_value);
+} else {
+    highest_value = a;
+    console.log("the highest value of the two is: " + highest_value);
+}
+
+
+
+
+
+//mothod 3
+let z = 7.25;
+console.log(z);
+const a = Math.round(z);
+console.log(a);
+const s = z - a;
+let highest_value;
+if (s > a - z) {
+	highest_value = a + s;
+    console.log("the highest value of the two is: " + highest_value);
+} else {
+	highest_value = z + -s;
+    console.log("the highest value of the two is: " + highest_value);
+}
+
+
+
+
+//method 4
+let z = 7.25;
+console.log(z);
+let a = z;
+if (a - Math.trunc(a) <= 0.4) {
+	a = Math.trunc(a);
+} else {
+	a = 1 - (a - Math.trunc(a)) + a;
+}
+console.log(a);
+let highest_value;
+if (z > a) { 
+	highest_value = z;
+    console.log("the highest value of the two is: " + highest_value);
+} else {
+    highest_value = a;
+    console.log("the highest value of the two is: " + highest_value);
+}
+
+
+
+
+//method 5
+let z = 7.25;
+console.log(z);
+let a = z;
+n = (a + "").split(".");
+i =(n.splice(-1, 1));
+let decimal = i[0];
+if (decimal.charAt(0) <= 4) {
+	a = Math.trunc(a);
+} else {
+	a = 1 - (a - Math.trunc(a)) + a;
+}
+console.log(a);
+let highest_value;
+if (z > a) { 
+	highest_value = z;
+    console.log("the highest value of the two is: " + highest_value);
+} else {
+    highest_value = a;
+    console.log("the highest value of the two is: " + highest_value);
+}
```

Comment:
Hi Omar, ignoring the indentation issues in this code, I see what you were trying to achieve by doing it in different ways. This is fine.

However, please make sure that in the future, by doing things this way, you shouldn't over-complicate your solution or get confused. I hope it doesn't happen with you but please be careful. Remember to always keep your code as simple as possible and fix small problems at a time. 🙂